### PR TITLE
Upgrade Babel to 7.0

### PR DESCRIPTION
**Pull request by nhunzaker**
This commit updates all babel packages to version 7.0.0

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,28 +1,21 @@
...
       - save_cache:
           paths:
             - node_modules
           key: v1-dependencies-{{ checksum "package.json" }}
-
-      # Test commands Start here
       - run: yarn lint
       - run: yarn test
+      - run: yarn build
+      - run: yarn build:example
```
Comment (dbernheisel) [inline .circleci/config.yml]: 👍

File: .prettierrc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "semi": false,
+  "singleQuote": true
+}
```
Comment (dbernheisel) [inline .prettierrc]: I've seen this in `package.json` before in other projects. Do you have a preference? I'm good with this change, just curious of thoughts.

File: package.json [hunk 3]
```diff
@@ -25,55 +25,43 @@
...
     "webpack-cli": "^3.1.0",
-    "webpack-dev-server": "^3.1.5"
+    "webpack-dev-server": "^3.1.8"
   },
   "dependencies": {
     "classnames": "~2.2.6",
-    "create-react-class": "^15.6.3",
-    "debounce": "~1.1.0",
-    "invariant": "~2.2.4",
-    "prop-types": "^15.6.2",
```
Comment (dbernheisel) [inline package.json]: :wave:

File: .prettierrc (hunk 2 above)
Comment (nhunzaker) [reply .prettierrc]: Related PR: 
https://github.com/vigetlabs/our-react-way/pull/80

**Comment by nhunzaker** [comment]
Thanks for the review, @dbernheisel!

**Pull request merged**